### PR TITLE
Add php/meterpreter/reverse_tcp KB

### PR DESCRIPTION
**Pull request by cbrnrd**
This addresses issue #7141 asking for KB for php/meterpreter/reverse_tcp.

### CODE REVIEW

File: documentation/modules/payload/php/meterpreter/reverse_tcp.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+The php/meterpreter/reverse_tcp is a staged payload used to gain meterpreter access to a compromised system. This is a unique payload in the Metasploit Framework because this payload is one of the only payloads that are used in RFI vulnerabilities in web apps. This module _can_ be cross platform, but the target needs to ba able to run php code.
+
+
+## Vulnerable Application
+
+  The Php Meterpreter is suitable for any system that supports PHP. For example, the module can be used against webservers which run PHP code for a website. OS X has PHP installed by default.
```
Comment (timwr) [inline documentation/modules/payload/php/meterpreter/reverse_tcp.md]: Php -> PHP

File: documentation/modules/payload/php/meterpreter/reverse_tcp.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+## Deploying /meterpreter/reverse_tcp
```
Comment (timwr) [inline documentation/modules/payload/php/meterpreter/reverse_tcp.md]: /meterpreter -> php/meterpreter

**Comment by thecarterb** [comment]
@timwr Thanks for pointing those out. Should be fixed now.

**Comment by thecarterb** [comment]
I'm going to separate this PR in to 2 PR's

**Pull request closed without merging**